src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json

```diff
@@ -8,5 +8,7 @@
     "sulu_media.information_taxonomy": "Informationen und Taxonomien",
     "sulu_media.upload_or_replace": "Bild hochladen / ersetzen",
     "sulu_media.media_selected_singular": "Medien Element ausgewählt",
-    "sulu_media.media_selected_plural": "Medien Elemente ausgewählt"
+    "sulu_media.media_selected_plural": "Medien Elemente ausgewählt",
+    "sulu_media.drop_files_to_upload": "Files hierher ziehen um hochzuladen",
+    "sulu_media.click_here_to_upload": "oder klicken um hinzuzufügen"
```

Comment:
Shouldn't we merge the previous two translations? What if a language doesn't allow to split it like that?